platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/eventhandling/ConsensusRoundHandler.java

```diff
@@ -111,6 +113,8 @@ public class ConsensusRoundHandler implements ConsensusRoundObserver, Clearable,
 
     private final RoundAppliedToStateConsumer roundAppliedToStateConsumer;
 
+    private final int roundsNonAncient;
```

Comment:
is this variable ever read anywhere?

Reply:
Yes, I was reading it it to get the minimum generation non-ancient. But since I've moved this data into `ConsensusRound`, I don't need to manually recompute it again. Removed.